.gitignore

```diff
@@ -1,71 +1,31 @@
-# Logs
-logs
-*.log
-npm-debug.log*
-yarn-debug.log*
-yarn-error.log*
-
-# Docs
-./docs/
-# Jetbrains
-.idea/
-
-# Runtime data
-pids
-*.pid
-*.seed
-*.pid.lock
-
-# Directory for instrumented libs generated by jscoverage/JSCover
-lib-cov
-
-# Coverage directory used by tools like istanbul
-coverage
-
-# nyc test coverage
-.nyc_output
-
-# Grunt intermediate storage (http://gruntjs.com/creating-plugins#storing-task-files)
-.grunt
+# See https://help.github.com/ignore-files/ for more about ignoring files.
 
-# Bower dependency directory (https://bower.io/)
-bower_components
+# dependencies
+/node_modules
 
-# node-waf configuration
-.lock-wscript
+# testing
+/coverage
 
-# Compiled binary addons (http://nodejs.org/api/addons.html)
-build/*.map
-build/*.json
-build/*.js
+# production
+/build
```

Comment:
Well actually you want to version the public folder because that contains you index.html and the service worker. CRA builds from the public folder into the build folder. Yeah it through me off the first time too